ocdstoucan/settings.py

```diff
@@ -141,29 +141,6 @@
 
 LANGUAGE_CODE = 'en-us'
 
-OCDS_TOUCAN_UPLOAD_OPTIONS = {
-    'maxNumOfFiles': os.getenv('OCDS_TOUCAN_MAXNUMFILES', 20),
-    'maxFileSize': os.getenv('OCDS_TOUCAN_MAXFILESIZE', 10000000)  # in bytes
-}
```

Comment:
Don't you think is better to keep Toucan's global settings in the settings file, for visibility at least? I think we can move the schema versions, but I would prefer to keep these here.

Reply:
Good point. Instead of setting OCDS_TOUCAN_UPLOAD_OPTIONS, let's instead set two constants for OCDS_TOUCAN_MAXNUMFILES and OCDS_TOUCAN_MAXFILESIZE.

A problem with the old code is that if you do:

```python
context = django_settings.OCDS_TOUCAN_UPLOAD_OPTIONS
context['performAction'] = '/{}/go/'.format(command)
```

You are actually changing the `OCDS_TOUCAN_UPLOAD_OPTIONS` dict; so if any other code uses that dict, it will have `performAction` set. By having two numeric constants instead, we can avoid that kind of issue.

Reply:
Are you happy to make that change?

Reply:
I see. Yes, I'll make the change.